clap_mangen/src/render.rs

```diff
@@ -29,7 +30,8 @@ pub(crate) fn description(roff: &mut Roff, cmd: &clap::Command) {
 }
 
 pub(crate) fn synopsis(roff: &mut Roff, cmd: &clap::Command) {
-    let mut line = vec![bold(cmd.get_name()), roman(" ")];
+    let name = cmd.clone().render_usage().to_string();
```

Comment:
`render_usage()` didn't create what I expected. Shouldn't it be `get_bin_name()`?

```suggestion
    let name = cmd.get_bin_name().unwrap_or_else(|| cmd.get_name());
```

Reply:
That looks like exactly what I wanted, cheers! Not sure how I didn't find it when I was working on this the other day, but there you go. I'll get that fixed up and see if I can get the rest of the tests passing :)